controllers/machine_controller.go

```diff
@@ -533,18 +534,18 @@ func (r *MachineReconciler) drainNode(ctx context.Context, cluster *clusterv1.Cl
 
 	if err := kubedrain.RunCordonOrUncordon(ctx, drainer, node, true); err != nil {
 		// Machine will be re-reconciled after a cordon failure.
-		log.Error(err, "Cordon failed")
-		return errors.Errorf("unable to cordon node %s: %v", node.Name, err)
+		log.Error(err, "Cordon failed, retry in 20s")
+		return ctrl.Result{RequeueAfter: 20 * time.Second}, nil
```

Comment:
Interesting we were not requeueing here before 🤔

Reply:
It felt strange to me, and it was discordant with the existing comment, so I added the requeue.

Reply:
The returned error was causing a re-reconciliation previously, any reason not to continue with the same behavior here now instead of using RequeueAfter here?

Reply:
Fixed